build.gradle

```diff
@@ -601,6 +601,12 @@ ext.DO_JCOV = Boolean.parseBoolean(JCOV)
 defineProperty("USE_CYGWIN", "true")
 ext.IS_USE_CYGWIN = Boolean.parseBoolean(USE_CYGWIN)
 
+// Specifies whether to run "chmod -R artifacts/sdk" as the last step of
+// copying the sdk to the artifacts dir. This is only done on Windows;
+// the flag is ignored on other platforms.
```

Comment:
a minor suggestion: mention JBS ticket for the context

Reply:
Normally, it is not a best practice to use the bug ID of the bug being fixed. In this case, because the fix is really a workaround for a GHA environment issue, it seems OK. The comment belongs in the GHA script, so I added it there.

Reply:
A good example of what should _not_ be done is something along the lines of ComboBoxListViewSkin:158:
```java
        // Fix for JDK-8115587. Additional code related to this bug is further below.
        this.listView.setManaged(false);
        getChildren().add(listView);
        // -- end of fix
```

Reply:
Exactly!  At the same time, `Declaration:160` might be an example of a suitable JBS reference.